templates/beta-post.adoc

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title: "TITLE"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/GITHUB_USERNAME
+author_github: https://github.com/GITHUB_USERNAME
+seo-title: TITLE - makes sure it ends with - OpenLiberty.io
+seo-description: DESCRIPTION
+blog_description: DESCRIPTION
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= TITLE
+AUTHOR_NAME <https://github.com/GITHUB_USERNAME>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+// // // // // // // //
+// Above:
+// Do not insert any blank lines between any of the lines above.
+// Do not remove or edit the variables on the lines beneath the author name.
+//
+// "open-graph-image" is set to OL logo, howver it can be changed to a different image or left blank in which case it will be set to the default  // Open Liberty logo
+//
+// Replace TITLE with the blog post title eg: MicroProfile 3.3 is now available on Open Liberty 20.0.0.4
+// Replace GITHUB_USERNAME with your GitHub username eg: lauracowen
+// Replace DESCRIPTION with a short summary (~60 words) of the release (a more succinct version of the first paragraph of the post).
+// Replace AUTHOR_NAME with your name as you'd like it to be displayed, eg: Laura Cowen
+//
+// If adding image into the post add :
+// -------------------------
+// [.img_border_light]
+// image::img/blog/FILE_NAME[IMAGE NAME/DESCRIPTION ,width=70%,align="center"]
+// -------------------------
+// "[.img_border_light]" = This adds a faint grey border around the image to make its edges sharper. Use it around screenshots but not            // around diagrams. Then double check how it looks.
+// Change "FILE_NAME" to the name of the image file. Also make sure to put the image into the right folder which is: img/blog
+// change the "IAMGE NAME/DESCRIPTION" to a couple words of what the image is
```

Comment:
IMAGE

Reply:
I wasn't sure wherever it should be Image name or description?

Reply:
I was just pointing out the spelling error

Reply:
I've noticed. Is it okay for me to leave it as that? Or is there something more appropriate than "IMAGE NAME/DESCRIPTION"

Reply:
Changed to "IMAGE CAPTION", I think that sounds a lot better.